templates/service_templates/redis.service.erb

```diff
@@ -1,17 +1,25 @@
 [Unit]
-Description=Advanced key-value store for <%= @title %>
+Description=Redis Advanced key-value store for instance <%= @title %>
```

Comment:
since there are so many changes in this template it would be awesome if it would be an epp template and not erb. Feel to convert it if you like it. We only require epp for completely new templates.

Reply:
Will do in a follow-up PR :) (see other comment)